compliance/COMPLIANCE.md

```diff
@@ -2,6 +2,21 @@
 
 This document outlines the compliance tests for the GoScript compiler, verifying its ability to correctly translate various Go language features into TypeScript.
 
+To run tests, you need to:
+
+* install [tsgo](https://github.com/microsoft/typescript-go) into `node_modules/.bin/tsgo` (e.g., create a symlink there to `~/go/bin/tsgo` after doing `go install` in `cmd/tsgo`)
```

Comment:
This shouldn't be necessary, as tsgo is in package.json and `yarn` automatically will link it to `./node_modules/.bin` you just need to run `yarn install`.

See: https://github.com/aperturerobotics/goscript/blob/502086f7d6b63846859956407bd13790b1925199/package.json#L92

Reply:
I updated the .md with additional information on master.

Reply:
I looked at the current docs and don't see instructions to run 'yarn install' -- probably everyone but me knows what this is, but I sure don't :)